rotkehlchen/exchanges/data_structures.py

```diff
@@ -534,6 +547,18 @@ class Loan(AccountingEventMixin):
     def get_timestamp(self) -> Timestamp:
         return self.close_time
 
+    def serialize(self) -> Dict[str, Any]:
```

Comment:
why do you need this?

Reply:
+1 here. Why? I guess same as the answer you gave for `MarginPosition`? If yes let's keep the convo there.